packages/gluestick/src/renderer/helpers/linkAssets.js

```diff
@@ -58,24 +75,43 @@ module.exports = function linkAssets(
     assets,
   );
   if (stylesHref) {
-    styleTags.push(
-      <link key={key++} rel="stylesheet" type="text/css" href={stylesHref} />,
-    );
+    if (config.GSConfig.inlineAllCss) {
+      const contents = await memoizedRead(
+        path.join(process.cwd(), 'build', stylesHref),
+      );
+      styleTags.push(
+        <style dangerouslySetInnerHTML={{ __html: contents.toString() }} />,
```

Comment:
I might be mistaken, but I think recent `react-helmet` allows `style`s with text as children.

Reply:
It does, but we're really low-level here (initial server render), and Helmet isn't really designed for order-dependent stuff like styles.